experimental/go-dep/image/Dockerfile-stack

```diff
@@ -0,0 +1,35 @@
+FROM golang:1
+
+ENV APPSODY_MOUNTS=/:/project/user-app
+
+ENV APPSODY_WATCH_DIR=/project/user-app
+ENV APPSODY_WATCH_IGNORE_DIR=/project/user-app/vendor
+ENV APPSODY_WATCH_REGEX="^.*.(go|toml)$"
+
+ENV APPSODY_PREP="bash /vendor.sh"
+
+ENV APPSODY_RUN="go run ."
+ENV APPSODY_RUN_ON_CHANGE="go run ."
+ENV APPSODY_RUN_KILL=false
+
+
+ENV APPSODY_TEST="go test ."
+ENV APPSODY_TEST_KILL=false
+
+COPY ./LICENSE /licenses/
+COPY ./project /project
+COPY ./config /config
+COPY ./vendor.sh /vendor.sh
+
+RUN chmod +x /vendor.sh
+
+WORKDIR /project/user-app
+
+RUN go get -u github.com/golang/dep/cmd/dep
+RUN go get -u github.com/go-delve/delve/cmd/dlv
```

Comment:
I understand this is for debugging, but I don't see it used anywhere and the README says debugging is not supported. Can this be removed?